README.md

```diff
@@ -958,8 +958,7 @@ Other Style Guides
     ```javascript
     // bad
     [1, 2, 3].map(number => {
-      const nextNumber = number + 1;
-      `A string containing the ${nextNumber}.`;
+      return `A string containing the ${number}.`;
```

Comment:
Unfortunately this is exactly why it’s a bad example, and it’s correct as is.

Reply:
Oh, ok then, got it. Thanks for the clarification.